llama_hub/airbyte_cdk/README.md

```diff
@@ -0,0 +1,42 @@
+# Airbyte CDK Loader
+
+The Airbyte CDK Loader is a shim for sources creating using the [Airbyte Python CDK](https://docs.airbyte.com/connector-development/cdk-python/). It allows you to load data from any Airbyte source into LlamaIndex.
+
+## Installation
+
+* Install llama_hub: `pip install llama_hub`
+* Install airbyte-cdk: `pip install airbyte-cdk`
+* Install a source via git (or implement your own): `pip install git+https://github.com/airbytehq/airbyte.git@master#egg=source_github&subdirectory=airbyte-integrations/connectors/source-github`
```

Comment:
(thought) wonder if we should gear these docs and this generic loader more as a way to use custom airbyte sources with llamaindex rather than installing our sources from github, since they should use our direct loaders instead for those.

Reply:
De-emphasized this is a bit.